package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -173,132 +166,316 @@ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
+  - only when water bridge is joined by less than **two** water.
+
+The analysis function can be written as::
+
+  def analysis(current, output, **kwargs):
+      '''This function defines how the counting of water bridge should be specified.
+
+        Parameters
+        ----------
+        current : list
+            The current water bridge being analysed is a list of hydrogen bonds from
+            selection 1 to selection 2.
+        output : dict
+            A dictionary where the key is the type of the water bridge and the value
+            is the number of this type of water bridge.
+            The output of this frame is the sum of all the values in this dictionary.
+      '''
+
+      # decompose the first hydrogen bond.
+      s1_index, to_index, (s1_resname, s1_resid, s1_name),
+      (to_resname, to_resid, to_name), dist, angle = current[0]
+      # decompose the last hydrogen bond.
+      from_index, s2_index, (from_resname, from_resid, from_name),
+      (s2_resname, s2_resid, s2_name), dist, angle = current[-1]
+
+      # only the residue name is ASP and the atom name is OD1,
+      if s2_resname == 'ASP' and s2_name == 'OD1':
+          # only if the order of water bridge is less than 2
+          if len(current) -1 < 2:
+              # extract all water molecules involved in the water bridge
+              # extract the first water from selection 1
+              s1_index, to_index, (s1_resname, s1_resid, s1_name),
+              (to_resname, to_resid, to_name), dist, angle = current[0]
+              key = (to_resname, to_resid)
+              output[key] = 1
+
+              # extract all the waters between selection 1 and selection 2
+              for hbond in current[1:-1]:
+                  # decompose the hydrogen bond.
+                  from_index, to_index, (from_resname, from_resid, from_name),
+                  (to_resname, to_resid, to_name), dist, angle = hbond
+                  # add first water
+                  key1 = (from_resname, from_resid)
+                  output[key1] = 1
+                  # add second water
+                  key2 = (to_resname, to_resid)
+                  output[key2] = 1
+
+              # extract the last water to selection 2
+              from_index, s2_index, (from_resname, from_resid, from_name),
+              (s2_resname, s2_resid, s2_name), dist, angle = current[-1]
+              key = (from_resname, from_resid)
+              output[key] = 1
+
+  w.count_by_time(analysis_func=analysis)
+
+Returns ::
+
+  [1, 0,]
+
+Classes
+-------
+
+.. autoclass:: WaterBridgeAnalysis
+   :members:
+
+   .. attribute:: timesteps
+
+      List of the times of each timestep. This can be used together with
+      :attr:`~WaterBridgeAnalysis.timeseries` to find the specific time point
+      of a water bridge existence.
 
-         import pandas as pd
-         df = pd.DataFrame.from_records(w.table)
 """
-from __future__ import absolute_import, division
```

Comment:
I'm pretty sure you should keep this.

Reply:
Please put it back.

Reply:
@orbeckst Sorry for the confusion. This line is in line 466 and I'm not sure why it isn't showing up in the diff (you can see it if you open the file).

Reply:
Ok, I guess since I'm replacing the original 
from __future__ import absolute_import, division
with
from __future__ import print_function, absolute_import, division
The diff is not picking it up.